### PR TITLE
Starred messages T13F2-48

### PR DESCRIPTION
**Pull request by taneja1995**

### CODE REVIEW

**Review by lalwanijayesh — CHANGES_REQUESTED** [review]
Can we please add some documentation for the new files?

File: controllers/StarController.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+export default class StarController implements StarControllerI{
+
+    private static starDao: StarDao = StarDao.getInstance();
+    private static starController: StarController | null = null;
+
+    public static getInstance = (app: Express): StarController => {
+        if (StarController.starController == null) {
+            StarController.starController = new StarController();
+            //app.get('/tuits/:tid/likes', StarController.starController.findAllUsersThatLikedTuit);
+            //app.get('/users/:uid/likes', StarController.starController.findAllTuitsLikedByUser);
```
Comment (lalwanijayesh) [inline controllers/StarController.ts]: Let's remove the commented code here?

Comment (taneja1995) [reply controllers/StarController.ts]: Removed the commented code

**Review by lalwanijayesh — APPROVED** [review]
Looks good! Thank you for making the changes, Ishita.

**Review by Shriya-Hukkeri — APPROVED** [review]
Looks good

**Pull request merged**